### PR TITLE
Copy changes for home page of NIAID clinical trials:

### PR DESCRIPTION
**Pull request by BedfordWest**
Link to Jira ticket if there is one:

### Environments
QA NIAID Clinical Data Trials (qa-niaid.planx-pla.net)

### Description of changes
Change some copy per request:
- Remove mailto link for data submission
- Add new paragraph with mailto link and copy
- Move the 2023 year over a word

### CODE REVIEW

**Review by paulineribeyre — DISMISSED** [review]

File: qa-niaid.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -13,8 +13,9 @@
...
+          "AccessClinicalData@NIAID can help support your Data Management Plan (DMP) to share clinical research data in compliance with the NIH 2023 Data Management and Sharing Policy.",
+	        "<p style='margin-left: 20px; font-style: italic;'>De-identified human clinical research and trials data can be submitted to accessclinicaldata@NIAID, which is a NIAID cloud-based, controlled access, secure data platform that enables sharing of and access to human clinical research and trials data sets and related metadata for the basic and clinical research community.  There is no cost to share and access data in this NIAID supported data platform.</p><br />",
+          "<p style='margin-left: 20px; font-style: italic;'>If you are interested in making your data available, please contact us at <a href='mailto:accessclinicaldatasupport@niaid.nih.gov'>accessclinicaldatasupport@niaid.nih.gov</a>.</p>",
```
Comment (frankliuao) [inline qa-niaid.planx-pla.net/portal/gitops.json:18]: I think Maria wanted the hyper link to be on "contact us". 
So: 
`please <a href='mailto:accessclinicaldatasupport@niaid.nih.gov'>contact us</a>.`

**Review by frankliuao — CHANGES_REQUESTED** [review]
Could you see the line comment

**Review by frankliuao — APPROVED** [review]
Changing texts on the front page to address Maria's request. Approved.

**Comment by frankliuao** [comment]
I think I'll just merge it without waiting for the CI to pass. This obviously has no potential harm to the website.

**Pull request merged**